commands/src/main/java/org/wildfly/extras/creaper/commands/ra/AddAdminObjectToRA.java

```diff
@@ -0,0 +1,87 @@
+package org.wildfly.extras.creaper.commands.ra;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Adds admin object to existing resource adapter
+ */
+public class AddAdminObjectToRA implements OnlineCommand {
+    private String poolName;
+    private String localJndiName;
+    private String className;
+    private String resourceAdapterId;
+    private boolean useJavaContext;
+    private Map<String, String> properties = new HashMap<String, String>();
+
+    private AddAdminObjectToRA(Builder builder) {
+        this.poolName = builder.poolName;
+        this.localJndiName = builder.localJndiName;
+        this.className = builder.className;
+        this.resourceAdapterId = builder.resourceAdapterId;
+        this.useJavaContext = builder.useJavaContext;
+        this.properties = builder.properties;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Operations ops = new Operations(ctx.client);
+        Address address = Address.subsystem("resource-adapters").and("resource-adapter", resourceAdapterId)
+                .and("admin-objects", poolName);
+        ops.add(address, Values.of("class-name", className)
+                .and("jndi-name", localJndiName).and("use-java-context", useJavaContext));
+        if (properties.size() > 0) {
+
+            Iterator it = properties.entrySet().iterator();
+            while (it.hasNext()) {
```

Comment:
Why not `for (Map.Entry<String, String> entry : properties.entrySet()) { ... }`?